grades/models.py

```diff
@@ -39,6 +39,7 @@ class FinalGrade(TimestampedModel, AuditableModel):
         default=FinalGradeStatus.PENDING,
         max_length=30,
     )
+    course_run_payed_on_edx = models.BooleanField(default=False)
```

Comment:
I also assume that we either have no or very few records in this table in production since this field isn't nullable (and hence PG will update all existing records)?